lib/index.js

```diff
@@ -14,50 +12,10 @@ const path = require("path");
  */
 
 module.exports = function initialize(command, args) {
-	const popArgs = args ? args.slice(2).pop() : null;
-	switch (command) {
-		case "init": {
-			const initPkgs = args.slice(2).length === 1 ? [] : [popArgs];
-			//eslint-disable-next-line
-			return require("./commands/init.js")(initPkgs);
-		}
-		case "migrate": {
-			const filePaths = args.slice(2).length === 1 ? [] : [popArgs];
-			if (!filePaths.length) {
-				throw new Error("Please specify a path to your webpack config");
-			}
-			const inputConfigPath = path.resolve(process.cwd(), filePaths[0]);
-			//eslint-disable-next-line
-			return require("./commands/migrate.js")(inputConfigPath, inputConfigPath);
-		}
-		case "add": {
-			//eslint-disable-next-line
-			return require("./commands/add")();
-		}
-		case "remove": {
-			//eslint-disable-next-line
-			return require("./commands/remove")();
-		}
-		case "update": {
-			return require("./commands/update")();
-		}
-		case "serve": {
-			return require("./commands/serve").serve();
-		}
-		case "make": {
-			return require("./commands/make")();
-		}
-		case "generate-loader": {
-			return require("./generate-loader/index.js")();
-		}
-		case "generate-plugin": {
-			return require("./generate-plugin/index.js")();
-		}
-		case "info": {
-			return require("./commands/info.js")();
-		}
-		default: {
-			throw new Error(`Unknown command ${command} found`);
-		}
+	if (!command) {
+		throw new Error(`Unknown command ${command} found`);
+	} else if (command === "serve") {
+		return require(`./commands/${command}`).serve();
 	}
+	return require(`./commands/${command}`)(...args);
```

Comment:
This would throw an error no file found. Are we OK with this? Or do we want to throw a better error message?

Reply:
check for `require` and then doing this would make sense

Reply:
Yep, we've defined all the commands in `webpack/bin/webpack.js`, rest will the webpack file take care of. What do you mean with check for require?

Reply:
`if(require(...))`

Reply:
Can't we put this `require` request in `try catch` block? and in catch block `throw new Error` stating that command not found, otherwise the error thrown will be that module was not found.